.vib/owncloud/goss/vars.yaml

```diff
@@ -0,0 +1,18 @@
+mariadb:
+  auth:
+    database: test_db_name
+    username: test_db_user
+    password: test_db_password
+externalDatabase:
+  port: 3306
```

Comment:
We are using the built-in MariaDB chart so this is not used. Although you are using this param to check for the used MariaDB, we should avoid it to avoid confusion. If there is no param for modifying the MariaDB port in OwnCloud's values we don't need to check for it.